### DISCUSSION TITLE
Platinen mit ESP32 S3

### DISCUSSION THREAD

**Discussion by sentinel1111** [hoylabs/OpenDTU-OnBattery#1161] [General]
Hallo zusammen,

Ich hoffe ich darf das hier posten,falls nicht lösche ich es wieder.
Anfang des Jahres habe ich mich an die Herstellung einer Platine gewagt.Mit der Zeit sind mehrere Versionen entstanden. 
Es sind keine professionellen Platinen, und es gibt bestimmt Verbesserungspotenzial,aber sie funktionieren seit Monaten bei mir und Bekannten problemlos.

Ich habe hier noch 5-6 Platinen teilbestückt mit ein Esp32 S3 N16R8 und müsste auch noch genügend Teile hier haben, um sie fertig zu
 Bestücken.
Bevor ich sie bei Kleinanzeige reinstelle, dachte ich frage zuerst hier, ob jemand was brauchen kann.

zur Ausstattung der Platine:

- Esp32 S3 N16R8 (kann über Usb programmiert werden)
- 3x ADUM1201 Ve.Direkt Ports
- Max3485 z.B Pylontech RS485 Bus
- SN65HVD230DR Pylontech CAN Bus
- I2C Oled Anschluss
- Cmt2300 (nach Wunsch alleine oder mit NRF2401)
- NRF2401 (nach Wunsch alleine oder mit CMT2300)
- verschiedene Led's für Statusanzeigen.

Diese Platinen sind übrig geblieben und ich plane nicht weitere zu bestellen. Ein Gehäuse habe ich
 auch dazu und kann bei Bedarf auch gedruckt werden. Es passt dazu aber nur ein
 2,42" Display.

Ein paar Bilder:
![DSC_7499](https://github.com/user-attachments/assets/4d966c64-b526-4ae0-b3a9-e762c766c656)
![DSC_7500](https://github.com/user-attachments/assets/73b38b3b-d868-484a-a583-85e112174a3c)
![DSC_7501](https://github.com/user-attachments/assets/9203648d-96a7-4f93-a4de-c3dc4f0c813c)
![DSC_7502](https://github.com/user-attachments/assets/bf35fa01-18fa-4597-a388-d4d5769b6465)
![DSC_7491](https://github.com/user-attachments/assets/43d32dcf-df4c-4ccc-a320-c008b3d5a28b)
![20240715_070419 1](https://github.com/user-attachments/assets/7e81846c-4eda-4ec2-98ef-add34748db85)

**Comment by Harisjems** [comment]
Ich hätte Interesse an eine mit NRF und CMT mit Gehäuse. 
Was hast du vorgestellt inkl.versand DE

**Reply by sentinel1111** [reply]
Hallo, 


Die Platine wo der ESP32 und der NRF gesteckt sind, in den grauen Gehäuse, mit Display 50 Eur plus 4,5 Eur Versand .

Die anderen Platinen:

Platine mit ESP32 ohne Funkmodul 25 Eur
Funkmodul CMT  5 Eur
Funkmodul NRF   5 Eur
2.42" Display      17 Eur
Gehäuse             15 Eur (muss auch eine Fräse dazu benutzen für die acryl "lichtleiter" )

Wer was haben will, kann mich auf "claudiuc et web punkt de" anschreiben.

**Comment by Harisjems** [comment]
Kurze Rückmeldung,
Platine ist eingerichtet, inkl. neu gedruckter Gehäuse. Macht einen wertigen Eindruck.
Es kann sein, dass ich mir das einbilde, aber der NRF-Empfänger ist um Welten besser als mein alter NRF mit ESP32 4 MB mit externer Antenne.

S3 mit 16 MB ist schon ordentlich flotter, kann auch Einbildung sein :-)

Danke dafür.

**Reply by sentinel1111** [reply]
Freut mich. Danke für die Rückmeldung.
